ruby/src/api.cc

```diff
@@ -136,11 +136,7 @@ namespace leatherman { namespace ruby {
         } else {
             LOG_INFO("ruby was already loaded.");
         }
-        try {
-            return unique_ptr<api>(new api(move(library)));
-        } catch (lth_lib::missing_import_exception& ex) {
-            throw ex;
-        }
+        return unique_ptr<api>(new api(move(library)));
```

Comment:
I'd lean towards dropping the overhead of unique_ptr. We can just do `return api(move(library));`, assign to `static api instance = create();`, and `return instance;`.